src/KeyboardShortcuts.elm

```diff
@@ -0,0 +1,108 @@
+module KeyboardShortcuts exposing (..)
+
+import Html exposing (Html, span, text)
+import Html.Attributes exposing (class)
+import Html.Events
+import Json.Decode as Decode
+import Keyboard.Event exposing (KeyboardEvent, decodeKeyboardEvent)
+
+
+type Shortcut
+    = Single String
+    | Sequence String String
+    | Combination String String
+
+
+
+-- HELPERS
+
+
+indexToShortcut : Int -> Maybe String
+indexToShortcut index =
+    let
+        n =
+            index + 1
+    in
+    if n > 9 then
```

Comment:
above 9 indexes can't be shortcuts (perhaps later we'll add an alternative, but I think its ok for now)

Reply:
Seems okay to me too.